src/org/jitsi/impl/neomedia/transform/DiscardTransformEngine.java

```diff
@@ -63,11 +63,16 @@ public RawPacket reverseTransform(RawPacket pkt)
                 = (pkt.getFlags() & Buffer.FLAG_DISCARD) == Buffer.FLAG_DISCARD;
 
             long ssrc = pkt.getSSRCAsLong();
-            ResumableStreamRewriter rewriter = ssrcToRewriter.get(ssrc);
-            if (rewriter == null)
+
+            ResumableStreamRewriter rewriter;
+            synchronized(ssrcToRewriter)
```

Comment:
Same thing here, the `ssrcToRewriter` object is accessed by a single thread at the time. We should be clear about that tho, to avoid confusion in the future.